src/universal/utils/getDemoAvatar.js

```diff
@@ -0,0 +1,201 @@
+const path = '/static/images/avatars/demo/'
```

Comment:
we'll wanna probably import these pictures & let webpack pick the url instead of using a string for the path. otherwise, these will all be coming from our server instead of the CDN

Reply:
I figured, but is it bad performance to import many just to randomly pick two? Or does webpack only determine the path, and then only the final objects get downloaded in the view?

Reply:
in all likelihood, webpack will automatically bundle them all together. to avoid that, we could use async imports or just put the files on the CDN & use that function to get the cdn url (can't think of the name of it right now...)